Source/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -257,6 +258,7 @@ - (void)setAccessibilityElements:(NSArray *)accessibilityElements
 {
   ASDisplayNodeAssertMainThread();
   _accessibilityElements = nil;
```

Comment:
It's odd that this setter doesn't consider the provided param and always set the ivar to nil. We probably should rename it to `resetAccessibilityElements` or `clearAccessibilityElements`?

Reply:
Yeah that sounds good, I wasn't sure if this was intended or we just haven't had a chance to implement this portion yet :/

Reply:
I *think* setting the provided param to `_accessibilityElements` is the right way to do.

@maicki: thoughts?

Reply:
Also, if the `_accessibilityElements` ivar is updated here, we may not even need a new `_customAccessibilityElements` ivar because setting a non-nil array here will cause the setter to return it (i.e it won't call `[viewNode accessibilityElements]`.